src/shared/use-blurred-field.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect, useRef } from 'react'
+import { useFormState } from 'react-final-form'
+
+export const useBlurredField = () => {
+    const previouslyActiveFieldRef = useRef(null)
+    const [blurredField, setBlurredField] = useState(null)
```

Comment:
Let's try to avoid using `setState`, since that will cause an additional re-render. See how a `usePrevious`-hook can be implemented eg. [here](https://usehooks.com/usePrevious/). I think that this can be used here. 

Basically:

```javascript
const previouslyActiveFieldRef = useRef(null)
const { active } = useFormState({ subscription: { active: true } })
useEffect(() => {
        if (active) { // not sure if this check is needed
            previouslyActiveFieldRef.current = active
        }
    }, [active])

// this runs before useEffect, so this contains previous value
return previouslyActiveFieldRef.current
```